### DISCUSSION TITLE
CSS rules Is not working for HtmlText Module

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#3384] [General]
Hi,

I’ve 5 HtmlText modules in a page. Each module has set of CSS rules. If I added CSS rules inside the <style> tag in the HtmlText module then it is working but if I added all those CSS rules in the theme CSS file then it is not applying styles at all (theme CSS rules are loading but not applying).

**Comment by sbwalker** [comment]
@Sharafudeen I have no idea why it is not working for you. I have added the following elements to an HtmlText module (to test the 2 different methods for applying CSS styles to an element ie. using either an id or class):

```
<div>Not Bold</div> 
<div id="testid">Test Bold</div> 
<div class="testclass">Test Bold</div> 
```

and I added the CSS to the Oqtane.Server\wwwroot\Themes\Oqtane.Themes.OqtaneTheme\Theme.css:

```
#testid {
    font-weight: bold;
}

.testclass {
    font-weight: bold;
}
```

and the result is:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/e1dcbe96-6b27-4765-9ee7-595b1e536795)
